images/kubeadm/Dockerfile

```diff
@@ -49,11 +48,6 @@ RUN mkdir -p /tmp/terraform/ && \
     ) && \
     rm -rf /tmp/terraform
 
-# TODO(pipejakob): Instead of fetching a specific precompiled version, get
```

Comment:
I *think* this is fine because we ship kubectl in the underlying image.

Reply:
@pipejakob @luxas

Reply:
hmm, this is the kubectl used when `e2e.test` execs out, right?
I'm not sure how `e2e.test` is populated inside of this image and/or if kubectl is copied over from somewhere in that process.
I guess we can try to merge this change and see if stuff breaks or not :wink:

Reply:
no, the tests should be using the kubectl from the current build under test, this kubectl shouldn't be used for much of anything (other tests in some other job not using kubetest??)

Reply:
let's go with this and see what happens!